src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js

```diff
@@ -8,15 +8,19 @@ import PropTypes from "prop-types";
 import OptionList from "./SelectOptions";
 
 const required = value => (value ? undefined : "Required");
-
+const composeValidators = (required, validator) => value => {
+  if (typeof validator === "undefined") return required(value);
+  return validator(value);
+};
```

Comment:
Maybe can we move all validators + composeValidators into one file?

Reply:
I've moved validators and composeValidators into one file.